new_course/1_programming_fundementals/loops.md

```diff
@@ -11,13 +11,13 @@ It will be beneficial for you to code along to test the examples as you work thr
 ## Learning Outcomes
 *Look through these now and then use them to test yourself after doing the assignment*
 
-* You understand what a loop is and why it's useful
-* You know how to use the `loop` loop in Ruby
-* You know how to use the `while` loop in Ruby
-* You know how to use the `for` loop in Ruby
-* You know how to use the `times` loop in Ruby.
-* You know how to use the `until` loop in Ruby.
-* You know how to use the `upto` and `downto` loops in Ruby
+* What a loop is and why it's useful?
```

Comment:
Nice change @AREEBAISHTIAQ, this is a much better format for the learning outcomes in this lesson.

Can we take this a step further and put them in a question format, for example: 
`What is a loop and why is it useful?`

I think it will read better, whats your thoughts?

Reply:
Hi @KevinMulhern , maybe i have already changed them in question format